pkg/ctl/get/addon.go

```diff
@@ -152,6 +152,12 @@ func addAddonSummaryTableColumns(printer *printers.TablePrinter) {
 	printer.AddColumn("CONFIGURATION VALUES", func(s addon.Summary) string {
 		return s.ConfigurationValues
 	})
+	printer.AddColumn("NAMESPACE", func(s addon.Summary) string {
```

Comment:
Should we update the `eksctl create addon` CLI to accept a custom namespace parameter as well?

Reply:
Sounds good, added CLI support for namespace to match upstream `aws eks` cli